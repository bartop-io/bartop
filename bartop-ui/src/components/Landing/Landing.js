import React from 'react';
import styled from 'styled-components';

import AuthButton from '../AuthButton/AuthButton';
import DrinksList from '../DrinksList/DrinksList';

const Wrapper = styled.div`
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`;

const Landing = () => (
  <Wrapper>
    <AuthButton />
    <DrinksList />
  </Wrapper>
);

export default Landing;
