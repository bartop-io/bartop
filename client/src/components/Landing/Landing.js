import React from 'react';
import styled from 'styled-components';
import { animations } from '../styleUtils';
import logo from '../../images/logo.svg';

const Wrapper = styled.div`
  text-align: center;
`;

const Header = styled.header`
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
`;

const Logo = styled.img`
  animation: ${animations.spin} infinite 20s linear;
  height: 80px;
`;

const Title = styled.h1`
  font-size: 1.5em;
`;

const Intro = styled.p`
  font-size: large;
`;

const Landing = () => (
  <Wrapper>
    <Header>
      <Logo src={logo} alt="logo" />
      <Title>Welcome to React</Title>
    </Header>
    <Intro>
      To get started, edit <code>src/Landing.js</code> and save to reload.
    </Intro>
  </Wrapper>
);

export default Landing;
