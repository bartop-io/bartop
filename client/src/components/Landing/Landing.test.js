import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Landing from './Landing';

it('matches the snapshot', () => {
  const wrapper = shallow(<Landing />);
  expect(wrapper).toMatchSnapshot();
});
