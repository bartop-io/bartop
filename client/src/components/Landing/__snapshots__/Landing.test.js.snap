// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
.c0 {
  text-align: center;
}

.c1 {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.c2 {
  -webkit-animation: iVXCSc infinite 20s linear;
  animation: iVXCSc infinite 20s linear;
  height: 80px;
}

.c3 {
  font-size: 1.5em;
}

.c4 {
  font-size: large;
}

<Landing>
  <styled.div>
    <div
      className="c0"
    >
      <styled.header>
        <header
          className="c1"
        >
          <styled.img
            alt="logo"
            src="logo.svg"
          >
            <img
              alt="logo"
              className="c2"
              src="logo.svg"
            />
          </styled.img>
          <styled.h1>
            <h1
              className="c3"
            >
              Welcome to React
            </h1>
          </styled.h1>
        </header>
      </styled.header>
      <styled.p>
        <p
          className="c4"
        >
          To get started, edit 
          <code>
            src/Landing.js
          </code>
           and save to reload.
        </p>
      </styled.p>
    </div>
  </styled.div>
</Landing>
`;
